src/adaptix/_internal/morphing/facade/retort.py

```diff
@@ -79,6 +79,7 @@ class FilledRetort(OperatingRetort, ABC):
         IsoFormatProvider(time),
         SecondsTimedeltaProvider(),
 
+        FlagByExactValueProvider(),
         EnumExactValueProvider(),  # it has higher priority than scalar types for Enum with mixins
```

Comment:
I think that using high-level factories here shouble become a standard (``flag_by_exact_value`` and ``enum_by_exact_value``)